README.md

```diff
@@ -37,6 +37,14 @@ SPAs are web apps that render dynamically at runtime entirely in the browser, us
 
 Follow [these steps](https://github.com/apple/swift-docc#using-docc-to-build-and-preview-documentation) to generate a documentation archive, set the path to your renderer and render locally your documentation using Swift-DocC-Render.
 
+## Rendering and building docs
+
+You need to have [DocC](https://swift.org/documentation/docc) installed, in order to preview and build documentation. Read [Getting Started with Swift](https://www.swift.org/getting-started/) to learn more.
+
+To preview the docs, run `npm run docs:preview`. This will spawn a preview server on http://localhost:8000/documentation/doccrender.
```

Comment:
```suggestion
To preview the docs, run `npm run docs:preview`. This will spawn a preview server on http://localhost:8080/documentation/doccrender.
```

I _think_ 8080 is the default port.

Reply:
Its 8000 for docc-preview, 8080 for vue-cli

Reply:
Oh whoops, you're right 😆